metrics-0.1.0.json

```diff
@@ -19,39 +19,54 @@
   "description": "This schema defines the structure and values for reporting Q/A metrics for projects.",
   "properties": {
     "commit": {
+      "description": "Commit-Hash pointing to the state of the codebase used for generating the metrics.",
```

Comment:
Would it make sense to add a field for the branch where the metrics where generated? E.g. `main` or `feature/xyz`? Or will it always be the main branch?

Reply:
A branch isn't a unique point in time a commit hash is. Also at specific points in time multiple branches can point to the same sha1 (commit hash).